scripts/ci-e2e-lib.sh

```diff
@@ -43,6 +43,13 @@ capi:buildDockerImages () {
 # k8s::prepareKindestImages checks all the e2e test variables representing a Kubernetes version,
 # and makes sure a corresponding kindest/node image is available locally.
 k8s::prepareKindestImages() {
+  if [ -n "${MANAGEMENT_KUBERNETES_VERSION:-}" ]; then
```

Comment:
What about
```suggestion
  if [ -n "${KUBERNETES_VERSION_MANAGEMENT:-}" ]; then
```
So we have all the Kubernetes version with the same prefix